### PR TITLE
chore: enhance documentation

### LINKED ISSUE #536: [DOCS] Make the steps a bulleted list in the "How does it work" section
### Description

We can make the section more appealing and understandable to the user by explaining the steps in bullet points.

### Screenshots

image

### Additional information

Kindly assign me this issue.

### PR DESCRIPTION
**Pull request by singodiyashubham87**
Fixes #536 

I've made the necessary changes in the markdown file, Now there will be bullet points instead of long paragraphs.

### CODE REVIEW

File: apps/docs/docs/overview.md [hunk 1]
```diff
@@ -10,7 +10,12 @@ Reduced.to is a modern web application designed to shorten link URLs, making the
 
 ### How does it work?
 
-Simply paste a long URL into the input field and click the "Shorten" button. The application will generate a shortened URL that you can copy and share with others. When a user clicks on the shortened URL, they will be redirected to the original long URL. There is also a dashboard for registered users to view their shortened URLs and track their performance.
+* User paste a long URL into the input field
+* After pasting the URL, User click the "Shorten" button. 
+* The application will generate a shortened URL
+* User can copy and share this shortened link with others.  
+* When the user clicks on the shortened URL, they will be redirected to the original long URL. 
+* There is also a dashboard for registered users to view their shortened URLs and track their performance.
```
Comment (origranot) [inline apps/docs/docs/overview.md:18]: Consider change the text according to my suggestion.

```suggestion
* User pastes a long URL in the input field.
* Clicks "Shorten" button.
* The app creates a shorter URL.
* User can copy and share it.
* Clicking the short URL redirects to the long one.
* Registered users access a dashboard to track their links.
```

**Review by origranot — CHANGES_REQUESTED** [review]
@singodiyashubham87 Please take a look at my suggestion and commnet.
Also I noticed the CI workflow has failed.

**Comment by singodiyashubham87** [comment]
@origranot I've no idea why is this check failing, Can you please guide me?

**Comment by iamjatinchauhan** [comment]
@singodiyashubham87 Try removing extra spaces from end of each point and replace * with - for order list.

**Comment by singodiyashubham87** [comment]
@iamjatinchauhan Thanks for your assistance, Appreciate it!

**Comment by singodiyashubham87** [comment]
@origranot All checks have been passed, Kindly merge the PR.

**Comment by singodiyashubham87** [comment]
@origranot Please merge this PR.

**Pull request closed without merging**